Utils/Dataflow/README

```diff
@@ -133,8 +133,34 @@ following requirements:
 
 [3] EOP, End-of-Process marker. Default value: '\0' (NULL)
 
+==================
+* 3. Stage types *
+==================
+There are three types of stages:
+
+1. Source stage - an initial stage in a dataflow pipeline that gets data
+from an external source (CDS, Rucio, AMI, etc.).
+
+2. Transform stage - these stages perform a set of operations for different
+cases on metadata and form output messages according to a particular pattern
+(e.g. select relevant metadata from a raw dataset, append metadata
+to a message, prepare messages for upload).
+
+2.1 Pre-sink stage - the last transform stage in a dataflow pipeline that
+prepares metadata for upload via sink stage; it awares about messages
```

Comment:
"... ; is is aware of messages' formatting rules. ..."

Reply:
@Evildoor, seems like you missed my reply to the similar comment of yours above:
 (https://github.com/PanDAWMS/dkb/pull/278#discussion_r304834350). Could you take a look and let me (and @anastasiakaida) know what do you think?

Reply:
I'd rather say that what's wrong here is the fact that I've made the comment and then (regardless of your reply's existence) forgot about it and made the same suggestion again. My bad.

Reply:
The conversation is not solved (I mean, the original comment of @Evildoor is still not addressed in the current version of the README).

Reply:
@mgolosova My bad. Fixed now.

@Evildoor Thank you!